buildbuddy.yaml

```diff
@@ -8,11 +8,18 @@ actions:
         branches:
           - "master"
     bazel_commands:
-      - test //... --config=workflows
+      - test //... --config=workflows --test_tag_filters=-performance,-webdriver
   - name: Benchmark
     triggers:
       push:
         branches:
           - "master"
     bazel_commands:
-      - test //enterprise/server/test/performance/cache:cache_test --config=workflows
+      - test //... --config=workflows --test_tag_filters=+performance
+  - name: Browser tests
```

Comment:
Should we add these to the PR workflow too?

Reply:
I'm fine with running these on PRs and since they only take ~15s to run, and should probably stay that fast even as we add more tests since they can be run in parallel (done). We can decide separately whether we want to make them required (i.e. make this a branch protection check).

Reply:
Yeah I'm cool them non-blocking to start - and it forces us to keep them fast.